ember_debug/general-debug.js

```diff
@@ -19,17 +19,18 @@ const GeneralDebug = EmberObject.extend(PortMixin, {
   // Keep an eye on https://github.com/ember-cli/ember-cli/issues/3045
   emberCliConfig: computed(function() {
     let config;
-    $('meta[name]').each(function() {
-      const meta = $(this);
-      let match = meta.attr('name').match(/environment$/);
+    let metas = document.querySelectorAll('meta[name]');
+    for (let i = 0; i < metas.length; i++) {
```

Comment:
to encourage best practice and avoid performance issues `metas.length` could be cached even if it's a small loop. 

```
for (let i = 0, j = metas.length; i < j; i++) {
```